dashboard/tests/test_dashboard.py

```diff
@@ -1016,6 +1016,9 @@ def test_dashboard_module_load(tmpdir):
 def test_dashboard_module_no_warnings(enable_test_module):
     # Disable log_once so we will get all warnings
     from ray.util import debug
+    import six
```

Comment:
is tis intentional left there?

Reply:
no lol. Thanks for catching it. No harto leave it though. But I will remove it